src/course_bot/essay.clj

```diff
@@ -213,7 +215,7 @@
 
 (defn assignreviewers-talk [db {token :token :as conf} essay-code]
   (let [cmd (str essay-code "assignreviewers")
-        help (str "for admin only")]
+        help (str (tr :essay/for-admin-only))]
```

Comment:
It should be `:essay/assignreviewers-help`. Also, look at the other commands in this file, they should be in the same style.

Reply:
Ow. Better to have -info suffix, like in other files.